server/user.go

```diff
@@ -309,3 +309,18 @@ func (p *Plugin) disconnectUser(instance Instance, user *User) (*Connection, err
 
 	return conn, nil
 }
+
+func (p *Plugin) GetJiraUserFromMentions(instanceID types.ID, mentions model.UserMentionMap, userKey string) *jira.User {
+	mentionUser, found := mentions[userKey]
```

Comment:
If I'm not mistaken, won't `userKey` here contain the `@` symbol, when passed in from `executeAssign`? Maybe we should truncate it here if it exists?